src/NzbDrone.Core/Organizer/FileNameBuilder.cs

```diff
@@ -380,6 +380,18 @@ private void AddMediaInfoTokens(Dictionary<string, Func<TokenMatch, string>> tok
             {
                 mediaInfoAudioLanguages = $"[{mediaInfoAudioLanguages}]";
             }
+            else
+            {
+                var lang = LanguageParser.ParseLanguages(movieFile.RelativePath).FirstOrDefault();
+                if (lang != null)
+                {
+                    var isoLang = IsoLanguages.Get(lang);
+                    if (isoLang != null)
+                    {
```

Comment:
Mediainfo languages come from mediainfo not from file naming parsing

Reply:
oh my god :( .gitignore anf build.sh does not belongs to this commit :( my mistake. its my first push request

Reply:
> Mediainfo languages come from mediainfo not from file naming parsing

i know, but sometimes sone non-correcr movies have UND audo languages :( and then bazzar tried download subtitle, becouse audio language is original czech, but described in mkv as UND and bazzar thinks that there is not czech language or subtitles and try to download one. in this case i can "hardcoded" language to filename

Reply:
You're saying Czech or Slovak is being treated as unknown lang? 

Parsing the filename and inaccurately passing file parsed audio as media info detected isn't the correct approach.  Pretty sure parsing already looks at the filename for languages elsewhere

Reply:
> You're saying Czech or Slovak is being treated as unknown lang?
> 
> Parsing the filename and inaccurately passing file parsed audio as media info detected isn't the correct approach. Pretty sure parsing already looks at the filename for languages elsewhere

no, sometimes some old, or non-correct release had BAD language in mkv. this tv show have CZECH language but in mkv set as UND. and is much faster rename file than encode every bad release
![image](https://user-images.githubusercontent.com/11536984/173207153-82f337b5-959e-4fb3-b9ea-0688556b6023.png)

Reply:
i deleted .gitignore and build.sh from this commit, is it ok? these two files does not belong to this commit, it was mistake

Reply:
oh my god :( i made bad PR [Fix: Add better czech slovak support for audio and datetime in UI] (https://github.com/Radarr/Radarr/pull/7353#) #7353 
to this PR belong only 1 commit [Fix: add better support for czech/slovak movie language and czech/slo…](https://github.com/Radarr/Radarr/pull/7353/commits/21beb93623937bf9e77d56d0716b80c61ffc110b)

can i fix it somehow please?

Reply:
> no, sometimes some old, or non-correct release had BAD language in mkv. this tv show have CZECH language but in mkv set as UND. and is much faster rename file than encode every bad release. THIS my change applied ONLY in situation when radarr does not detect language, like in this situation without my modification movie/tvshow set as ENGLISH (and thats not true) and barazz tried download subtitles and radarr tried find another czech release. ![image](https://user-images.githubusercontent.com/11536984/173207153-82f337b5-959e-4fb3-b9ea-0688556b6023.png)

what's ffprobe think it is?

/path/to/radarr/binaries/ffprobe -v quiet -print_format json -show_format -sexagesimal -show_streams  -i /path/to/file

Reply:
```
ffprobe -v quiet -print_format json -show_format -sexagesimal -show_streams -i /******/test-old-movie.czech.mkv 
{
    "streams": [
        {
            "index": 0,
            "codec_name": "mpeg4",
            "codec_long_name": "MPEG-4 part 2",
            "profile": "Advanced Simple Profile",
            "codec_type": "video",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "width": 608,
            "height": 448,
            "coded_width": 608,
            "coded_height": 448,
            "closed_captions": 0,
            "film_grain": 0,
            "has_b_frames": 1,
            "sample_aspect_ratio": "1:1",
            "display_aspect_ratio": "19:14",
            "pix_fmt": "yuv420p",
            "level": 5,
            "chroma_location": "left",
            "refs": 1,
            "quarter_sample": "false",
            "divx_packed": "false",
            "r_frame_rate": "25/1",
            "avg_frame_rate": "25/1",
            "time_base": "1/1000",
            "start_pts": 40,
            "start_time": "0:00:00.040000",
            "extradata_size": 60,
            "disposition": {
                "default": 1,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "language": "eng",
                "HANDLER_NAME": "Video Media Handler",
                "BPS": "1254457",
                "DURATION": "00:51:52.120000000",
                "NUMBER_OF_FRAMES": "77803",
                "NUMBER_OF_BYTES": "488002724",
                "_STATISTICS_WRITING_APP": "mkvmerge v67.0.0 ('Under Stars') 64-bit",
                "_STATISTICS_WRITING_DATE_UTC": "2022-06-11 23:47:14",
                "_STATISTICS_TAGS": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES"
            }
        },
        {
            "index": 1,
            "codec_name": "aac",
            "codec_long_name": "AAC (Advanced Audio Coding)",
            "profile": "LC",
            "codec_type": "audio",
            "codec_tag_string": "[0][0][0][0]",
            "codec_tag": "0x0000",
            "sample_fmt": "fltp",
            "sample_rate": "44100",
            "channels": 2,
            "channel_layout": "stereo",
            "bits_per_sample": 0,
            "r_frame_rate": "0/0",
            "avg_frame_rate": "0/0",
            "time_base": "1/1000",
            "start_pts": 0,
            "start_time": "0:00:00.000000",
            "extradata_size": 2,
            "disposition": {
                "default": 1,
                "dub": 0,
                "original": 0,
                "comment": 0,
                "lyrics": 0,
                "karaoke": 0,
                "forced": 0,
                "hearing_impaired": 0,
                "visual_impaired": 0,
                "clean_effects": 0,
                "attached_pic": 0,
                "timed_thumbnails": 0,
                "captions": 0,
                "descriptions": 0,
                "metadata": 0,
                "dependent": 0,
                "still_image": 0
            },
            "tags": {
                "HANDLER_NAME": "Sound Media Handler",
                "BPS": "80686",
                "DURATION": "00:51:52.054000000",
                "NUMBER_OF_FRAMES": "134025",
                "NUMBER_OF_BYTES": "31387682",
                "_STATISTICS_WRITING_APP": "mkvmerge v67.0.0 ('Under Stars') 64-bit",
                "_STATISTICS_WRITING_DATE_UTC": "2022-06-11 23:47:14",
                "_STATISTICS_TAGS": "BPS DURATION NUMBER_OF_FRAMES NUMBER_OF_BYTES"
            }
        }
    ],
    "format": {
        "filename": "/******/test-old-movie.czech.mkv",
        "nb_streams": 2,
        "nb_programs": 0,
        "format_name": "matroska,webm",
        "format_long_name": "Matroska / WebM",
        "start_time": "0:00:00.000000",
        "duration": "0:51:52.160000",
        "size": "520240848",
        "bit_rate": "1337311",
        "probe_score": 100,
        "tags": {
            "COMPATIBLE_BRANDS": "mp42isomndss",
            "creation_time": "2022-06-11T23:47:14.000000Z",
            "MAJOR_BRAND": "ndss",
            "MINOR_VERSION": "0",
            "ENCODER": "Lavf57.56.101"
        }
    }
}
```

Reply:
someone didnot set LANGUEAGE for track. its "normal" for very old release or "home made rips"
![image](https://user-images.githubusercontent.com/11536984/173208735-509dbc10-b0ba-47b2-9191-4f65432b7a89.png)

Reply:
i made corrected PR Fix: add better support for czech/slovak movie language and czech/slo… #7354
with only one commit with correct files.
if its better